api/src/main/java/org/dicekeys/api/DerivationRecipeTemplates.kt

```diff
@@ -30,3 +30,6 @@ val derivationRecipeTemplates: Array<DerivationRecipe> = arrayOf(
 	DerivationRecipe(DerivationOptions.Type.Password, "LastPass", """{"allow":[{"host":"*.lastpass.com"}]}"""),
 	DerivationRecipe(DerivationOptions.Type.Password, "Microsoft", """{"allow":[{"host":"*.microsoft.com"},{"host":"*.live.com"}]}""")  
 )
+
+
+val seedSecurityKeyRecipeTemplate = DerivationRecipe(DerivationOptions.Type.Secret, "Seed Security Key", """{"purpose":"seedSecurityKey"}""")
```

Comment:
That recipe is used to create a seed for a soloKey.  Since that file is a list of templates, I think it's fine located there.  I suppose the question is is it being used as a template so that soloKeys can be seeded with different sequence numbers?